### ISSUE TITLE
mongodb-memory-server-global-3.6 downloads two mongos

### ISSUE THREAD

**Issue by helt** [nodkz/mongodb-memory-server#174]
seems like `mongodb-memory-server-global-3.6` does more work than neccessary.
package.json:
```
    "devDependencies": {
        "@types/mongodb-memory-server": "1.8.0",
        "mongodb-memory-server-global-3.6": "5.0.4",
    },

```

on my dev machine
```
$ npm install

> mongodb-memory-server@5.0.4 postinstall C:\project\package\node_modules\mongodb-memory-server
> node ./postinstall.js

mongodb-memory-server: checking MongoDB binaries cache...
mongodb-memory-server: binary path is C:\user\home\.cache\mongodb-binaries\4.0.3\mongod.exe

> mongodb-memory-server-global-3.6@5.0.4 postinstall C:\project\package\node_modules\mongodb-memory-server-global-3.6
> node ./postinstall.js

mongodb-memory-server: checking MongoDB binaries cache...
mongodb-memory-server: binary path is C:\user\home\.cache\mongodb-binaries\3.6.12\mongod.exe

> nodemon@1.18.7 postinstall C:\project\package\node_modules\nodemon
> node bin/postinstall || exit 0

```

in gitlab ci:
```
[INFO] --- frontend-maven-plugin:1.6:npm (npm-install) @ project-artifact-id ---
[INFO] Running 'npm install' in /builds/namespace/project/package
[INFO] 
[INFO] > mongodb-memory-server@5.0.4 postinstall /builds/namespace/project/package/node_modules/mongodb-memory-server
[INFO] > node ./postinstall.js
[INFO] 
[INFO] mongodb-memory-server: checking MongoDB binaries cache...
[INFO] Downloading MongoDB 4.0.3: 0 % (0mb / 79.4mb)
[INFO] mongodb-memory-server: binary path is /builds/namespace/project/package/node_modules/.cache/mongodb-memory-server/mongodb-binaries/4.0.3/mongod
[INFO] > mongodb-memory-server-global-3.6@5.0.4 postinstall /builds/namespace/project/package/node_modules/mongodb-memory-server-global-3.6
[INFO] > node ./postinstall.js
[INFO] 
[INFO] mongodb-memory-server: checking MongoDB binaries cache...
[INFO] Downloading MongoDB 3.6.12: 0 % (0mb / 114.2mb)
[INFO] mongodb-memory-server: binary path is /root/.cache/mongodb-binaries/3.6.12/mongod
```

**Comment by hasezoey** [comment]
this shouldnt happen, is it installed multiple times across some packages?

**Comment by hasezoey** [comment]
from what i can tell, this was an error because the `global-x.x` versions depended on `mongodb-memory-server` (not core), which has its own postinstall, this got fixed in some commit

closing because this got fixed some time ago

**Comment by hasezoey** [comment]
closing because fixed since `7.0.0-beta.6`
